<launch>
    <!-- launch specification file to get parameters for the task -->
    <include file="$(find uniform_object_rearrangement)/launch/task_specification.launch"></include>

    <!-- execute task plan -->
    <!-- arg (1)#object (2)instance_id -->
    <node pkg="uniform_object_rearrangement" type="ExecuteTaskPlan.py" name="execute_task_plan" 
        args="$(arg execute_task_plan)" output="screen" required="True" />

    <node pkg="uniform_object_rearrangement" type="PybulletExecutionScene.py" name="pybullet_execution_scene"
		    output="screen" required="True" />
   
</launch>